src/main/java/com/linkedin/kmf/KafkaMonitor.java

```diff
@@ -33,12 +42,15 @@
 public class KafkaMonitor {
   private static final Logger LOG = LoggerFactory.getLogger(KafkaMonitor.class);
   public static final String CLASS_NAME_CONFIG = "class.name";
+  private static final String METRIC_GROUP_NAME = "kafka-monitor";
+  private static final String JMX_PREFIX = "kmf";
```

Comment:
We changed the project name from kafka-monitor-framework but the package name is still `com.linkedin.kmf` and the JMX prefix used by all existing metrics are still `kmf.services`. I don't think we will likely change the existing package name and JMX prefix. Thus it may be OK to use `kmf` in the future package name and the JMX prefix so that they are consistent. I am not strong about the JMX prefix name here. Do you prefer me to change it to `kafka-monitor`?